tutorwordpress/patches/local-docker-compose-dev-services

```diff
@@ -0,0 +1,15 @@
+wordpress:
+  stdin_open: true
+  tty: true
+  environment:
+    WORDPRESS_DEBUG: {% if WORDPRESS_DEBUG %}1{% else %}0{% endif %}
```

Comment:
I suggest always setting this value to 1 in dev.

Reply:
You're right. It was hardcoded with 1 but I got a request from that "Probably we want to be able to edit variables like WP_DEBUG". I guess it's not a bad thing to let folks be able to switch this in their local env.

Reply:
I thought it was valuable to be able to change that setting to test a not debug WordPress page locally, but reviewing the `tutorwordpress/patches/local-docker-compose-services`, we could test it running the local mode of tutor, and that's it.

I don't have a strong opinion about this. Feel free to follow the suggestion here @CodeWithEmad.

Reply:
@MaferMazu I'll remove the `WORDPRESS_DEBUG` flag from the plugin and set to 1 in `local-docker-compose-dev-services`.